packages/turf-clusters-distance/test.js

```diff
@@ -63,35 +63,41 @@ test('clusters -- prevent input mutation', t => {
 });
 
 test('clusters -- translate properties', t => {
-    t.equal(clustersDistance(points, 2, 'kilometers', 1).points.features[0].properties.foo, 'bar');
+    t.equal(clustersDistance(points, 2, 'kilometers', 1).features[0].properties.foo, 'bar');
     t.end();
 });
 
 // style result
 function colorize(clustered) {
-    let count = featureReduce(clustered.points, (count, point) => Math.max(count, point.properties.cluster || 0), 1) + 1;
+    const count = featureReduce(clustered, (count, point) => Math.max(count, point.properties.cluster || 0), 1) + 1;
     const colours = chromatism.adjacent(360 / count, count, '#0000FF').hex;
     const points = [];
 
-    featureEach(clustered.points, function (point) {
-        const color = colours[point.properties.cluster];
-        point.properties['marker-color'] = color;
-        point.properties['marker-size'] = 'small';
-        points.push(point);
-    });
-    featureEach(clustered.centroids, function (centroid) {
-        const color = chromatism.brightness(-25, colours[centroid.properties.cluster]).hex;
-        centroid.properties['marker-color'] = color;
-        centroid.properties['marker-symbol'] = 'star';
-        centroid.properties['marker-size'] = 'large';
-        points.push(centroid);
-    });
-    featureEach(clustered.noise, function (point) {
-        point.properties['marker-color'] = '#AEAEAE';
-        point.properties['marker-symbol'] = 'circle-stroked';
-        point.properties['marker-size'] = 'medium';
-        points.push(point);
+    featureEach(clustered, function (point) {
```

Comment:
lol Yep! Looks good.

I really like this Switch statement, makes it really easy to control those clustered points.